pycbc/workflow/coincidence.py

```diff
@@ -161,6 +161,25 @@ def create_node(self, coinc_files, ifos, tags=None):
         node.new_output_file_opt(seg, '.hdf', '--output-file', tags=tags)
         return node
 
+class PyCBCMultiifoStatMapInjExecutable(Executable):
```

Comment:
I think this need not be duplicated as the options seem to be same. Is there any reason not to use the existing class, instead of adding a new one? If you are just wanting to add an alias, you can do. 
```
class PyCBCMultiifoStatMapExecutable(Executable)
    pass
```

Reply:
Actually, if you check carefully, as all multiifo executables, I take the combination of ifos as an argument, (ifos) that is extra compared to the previous ones.